src/server/renderer.js

```diff
@@ -44,14 +44,14 @@ export default (projectConfig, projectToolsConfig) => {
           console.error('ROUTER ERROR:', pretty.render(error));
           res.status(500);
         } else if (renderProps) {
-          loadPropsOnServer(renderProps, (err, asyncProps) => {
+          loadOnServer(renderProps, store).then(() => {
```

Comment:
And better do it like follows, in case you accidentally didn't know (see [here](https://github.com/petkaantonov/bluebird/wiki/Promise-anti-patterns#the-thensuccess-fail-anti-pattern)) 
```
.then(() => {})
.catch((err) => {})
```